test/unit/local/steps/incomplete_fixer.js

```diff
@@ -263,5 +263,40 @@ describe('core/local/steps/incomplete_fixer', () => {
         ])
       })
     })
+
+    describe('file renamed and then renamed back to its previous name', () => {
+      it('results in no events at all', async () => {
+        const src = 'src'
+        const dst = 'dst'
+        const firstRenamedEvent = builders.event()
+          .kind('file')
+          .action('renamed')
+          .oldPath(src)
+          .path(dst)
+          .incomplete()
+          .build()
+        const secondRenamedEvent = builders.event()
+          .kind('file')
+          .action('renamed')
+          .oldPath(dst)
+          .path(src)
+          .build()
+        const incompletes = []
+        const outputBatches = []
+
+        // Create actual `src` file
+        await syncDir.makeTree(['src'])
```

Comment:
I believe `makeTree` is nice when we're mixing dirs/subdirs/files, but here it's easy to wonder whether `src` should actually be `src/`.

Maybe introduce `syncDir.ensureFile('src')`?
Which would probably make the comment useless :wink: